diff -ru intel-ipsec-mb-0.54~/Makefile intel-ipsec-mb-0.54/Makefile
--- intel-ipsec-mb-0.54~/Makefile	2020-04-14 10:13:02.000000000 -0700
+++ intel-ipsec-mb-0.54/Makefile	2020-08-07 15:07:57.359339304 -0700
@@ -39,7 +39,7 @@
 SO_VERSION = $(shell echo $(VERSION) | cut -d. -f1)
 
 PREFIX ?= /usr
-LIB_INSTALL_DIR ?= $(PREFIX)/lib
+LIB_INSTALL_DIR ?= $(PREFIX)/lib64
 HDR_DIR ?= $(PREFIX)/include
 MAN_DIR ?= $(PREFIX)/man/man7
 MAN1 = libipsec-mb.7
@@ -538,15 +538,15 @@
 
 .PHONY: install
 install: $(LIB_DIR)/$(LIBNAME)
-	install -d $(HDR_DIR)
-	install -m 0644 $(IMB_HDR) $(HDR_DIR)
-	install -d $(LIB_INSTALL_DIR)
-	install -s -m $(LIBPERM) $(LIB_DIR)/$(LIBNAME) $(LIB_INSTALL_DIR)
-	install -d $(MAN_DIR)
-	install -m 0444 $(MAN1) $(MAN_DIR)
-	install -m 0444 $(MAN2) $(MAN_DIR)
+	install -d $(DESTDIR)$(HDR_DIR)
+	install -m 0644 $(IMB_HDR) $(DESTDIR)$(HDR_DIR)
+	install -d $(DESTDIR)$(LIB_INSTALL_DIR)
+	install -s -m $(LIBPERM) $(LIBNAME) $(DESTDIR)$(LIB_INSTALL_DIR)
+	install -d $(DESTDIR)$(MAN_DIR)
+	install -m 0444 $(MAN1) $(DESTDIR)$(MAN_DIR)
+	install -m 0444 $(MAN2) $(DESTDIR)$(MAN_DIR)
 ifeq ($(SHARED),y)
-	cd $(LIB_INSTALL_DIR); \
+	cd $(DESTDIR)$(LIB_INSTALL_DIR); \
 		ln -f -s $(LIB).so.$(VERSION) $(LIB).so.$(SO_VERSION); \
 		ln -f -s $(LIB).so.$(SO_VERSION) $(LIB).so
 ifneq ($(NOLDCONFIG),y)
