--- intel-ipsec-mb-0.52/Makefile~	2018-12-03 11:58:25.000000000 +0000
+++ intel-ipsec-mb-0.52/Makefile	2019-10-23 14:37:05.362655871 +0000
@@ -39,7 +39,7 @@
 SO_VERSION = $(shell echo $(VERSION) | cut -d. -f1)
 
 PREFIX ?= /usr
-LIB_INSTALL_DIR ?= $(PREFIX)/lib
+LIB_INSTALL_DIR ?= $(PREFIX)/lib64
 HDR_DIR ?= $(PREFIX)/include
 MAN_DIR ?= $(PREFIX)/man/man7
 MAN1 = libipsec-mb.7
@@ -304,13 +304,13 @@
 
 .PHONY: install
 install: $(LIBNAME)
-	install -d $(HDR_DIR)
-	install -m 0644 $(IMB_HDR) $(HDR_DIR)
-	install -d $(LIB_INSTALL_DIR)
-	install -s -m $(LIBPERM) $(LIBNAME) $(LIB_INSTALL_DIR)
-	install -d $(MAN_DIR)
-	install -m 0444 $(MAN1) $(MAN_DIR)
-	install -m 0444 $(MAN2) $(MAN_DIR)
+	install -d $(DESTDIR)$(HDR_DIR)
+	install -m 0644 $(IMB_HDR) $(DESTDIR)$(HDR_DIR)
+	install -d $(DESTDIR)$(LIB_INSTALL_DIR)
+	install -s -m $(LIBPERM) $(LIBNAME) $(DESTDIR)$(LIB_INSTALL_DIR)
+	install -d $(DESTDIR)$(MAN_DIR)
+	install -m 0444 $(MAN1) $(DESTDIR)$(MAN_DIR)
+	install -m 0444 $(MAN2) $(DESTDIR)$(MAN_DIR)
 ifeq ($(SHARED),y)
 	cd $(LIB_INSTALL_DIR); \
 		ln -f -s $(LIB).so.$(VERSION) $(LIB).so.$(SO_VERSION); \
--- intel-ipsec-mb-0.52/Makefile~	2019-10-23 14:37:05.000000000 +0000
+++ intel-ipsec-mb-0.52/Makefile	2019-10-23 14:38:42.712654515 +0000
@@ -312,7 +312,7 @@
 	install -m 0444 $(MAN1) $(DESTDIR)$(MAN_DIR)
 	install -m 0444 $(MAN2) $(DESTDIR)$(MAN_DIR)
 ifeq ($(SHARED),y)
-	cd $(LIB_INSTALL_DIR); \
+	cd $(DESTDIR)$(LIB_INSTALL_DIR); \
 		ln -f -s $(LIB).so.$(VERSION) $(LIB).so.$(SO_VERSION); \
 		ln -f -s $(LIB).so.$(SO_VERSION) $(LIB).so
 ifneq ($(NOLDCONFIG),y)
--- intel-ipsec-mb-0.52/Makefile~	2019-10-23 14:38:42.000000000 +0000
+++ intel-ipsec-mb-0.52/Makefile	2019-10-23 14:41:11.883652438 +0000
@@ -315,9 +315,6 @@
 	cd $(DESTDIR)$(LIB_INSTALL_DIR); \
 		ln -f -s $(LIB).so.$(VERSION) $(LIB).so.$(SO_VERSION); \
 		ln -f -s $(LIB).so.$(SO_VERSION) $(LIB).so
-ifneq ($(NOLDCONFIG),y)
-	ldconfig
-endif
 endif
 
 .PHONY: uninstall
